app/components/Article/article.css

```diff
@@ -202,4 +205,19 @@ article .banner h3 .title {
   article {
     margin: 0;
   }
+
+  article .link-popup {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    background-color: #ff000033;
+    overflow: scroll;
+  }
+  article .link-popup .footnote {
+    margin: auto;
+    background-color: white;
```

Comment:
variable!

Reply:
I chose these specially for you. i.e. so that you'd get annoyed with it and change them :D